ultravox/tools/infer_tool.py

```diff
@@ -91,7 +91,7 @@ def run_tui(
         print(f"--- Sample {index} ---")
     messages = sample.messages
     transcript = f' ["{sample.audio_transcript}"]' if sample.audio_transcript else ""
-    print(f"Q: {messages[0]['content']}{transcript}")
+    print(f"Q: {messages[-2]['content']}{transcript}")  # TODO: change?
```

Comment:
what's the issue here?

Reply:
This one is probably okay. I left the TODOs mainly so that I can come back and verify that.